server/rest/src/main/java/org/infinispan/rest/CacheEntryInputStream.java

```diff
@@ -207,7 +208,13 @@ private void loadMetadata() {
          InternalCacheEntry ice = (InternalCacheEntry) currentEntry;
          // add metadata
          long lifespan = ice.getLifespan();
+         if (lifespan > -1) {
```

Comment:
So I must admit this seems a bit odd to me. Why even have the created and last used if we are converting the lifespan and max idle to seconds? Should we not convert all of them? Or maybe we don't even need created or last used?

Reply:
the rest api works in seconds for maxidle and lifespan
https://github.com/infinispan/infinispan/blob/4e9596fe46a6118f4b3e988baa0b4758377a688f/server/rest/src/main/java/org/infinispan/rest/NettyRestResponse.java#L194

Reply:
the other parameters are translated to dated from the long on the headers

Reply:
and for put entry for example we send timeToLiveSeconds
https://infinispan.org/docs/dev/titles/rest/rest.html#rest_v2_add_entries

Reply:
@gustavonalle here

Reply:
@wburns since gustavo is not coming until next year, could we move forward this please? I need this fix for the console because other wise data is not consistent. I can do conversion directly from the console, but it's very weird to me to work on seconds everywhere in the REST api but not here. I was disturbed myself

Reply:
I'm planning to implement the CLI operation on this as well

Reply:
Yeah, I understand we use seconds for REST. I was more asking because we have a Metadata class and normally metadata is in ms internally. Maybe just change the instance variables to `lifespanSeconds` instead?